pkg/operator/cluster/ceph/mon/health.go

```diff
@@ -156,6 +156,12 @@ func (c *Cluster) checkHealth() error {
 		return err
 	}
 
+	// create/start new mons when there are less mons
+	if len(status.MonMap.Mons) < c.Size {
```

Comment:
You need to restart the operator for that. With that change you don't need to restart the operator anymore to create more mons if you "lost" a mon.

Reply:
seems like this would require that someone remove the mon from the monmap, for example through the toolbox. Just trying to understand how this issue was hit.